db/migrate/20170609195100_remove_codelocation_status_default.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveCodelocationStatusDefault < ActiveRecord::Migration
+  def self.up
+    change_column_default :code_locations, :status, nil
```

Comment:
I apologize, I didn't see the description. IMHO, i would prefer setting a new value to status field instead of nil.